### PR TITLE
[doc] Adjust the logic before and after, add a small optimization when installing manager

### LINKED ISSUE #13922: [Doc] This part of the document needs to be optimized when installing manager
### What issue do you find in Pulsar docs?

The following URLs show content that needs to be optimized：
https://pulsar.apache.org/docs/zh-CN/administration-pulsar-manager/#set-administrator-account-and-password

1）Method 1: Use the command line tool
The URL is error： https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/apache-pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz
The right URL is： https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz

2）You can Install manager use Docker or use Method1（Not Docker），the two are side by side。

### What is your suggestion?

Modify the error。
The revised ones are more logical.

### Do you have any references?

no

### Would you like to fix this issue?

yes

### Note

- [X] I have researched my question.

### PR DESCRIPTION
**Pull request by zhaoyajun2009**
### Motivation
Adjust the logic before and after, add a small optimization

Fixes #13922 

### Modifications

1）add 1 notes
2）add method2 of deploy manager
3) Adjust ‘Set administrator account and password’ to the back

### Does this pull request potentially affect one of the following parts:
  - Anything that affects deployment: (yes )

### Documentation
  
-  

- [x] `doc`

### CODE REVIEW

**Comment by zhaoyajun2009** [comment]
Please @tuteng help review , thanks.

**Comment by Anonymitaet** [comment]
Hi @tuteng can you review this PR from the technical perspective? Then writers can review it from the tech writing perspective, thanks.

**Review by tuteng — APPROVED** [review]

File: site2/docs/administration-pulsar-manager.md [hunk 1]
```diff
@@ -6,36 +6,29 @@ sidebar_label: Pulsar Manager
...
 Pulsar Manager is a web-based GUI management and monitoring tool that helps administrators and users manage and monitor tenants, namespaces, topics, subscriptions, brokers, clusters, and so on, and supports dynamic configuration of multiple environments.
 
-> **Note**   
+> **Note**
 > If you are monitoring your current stats with [Pulsar dashboard](administration-dashboard.md), we recommend you use Pulsar Manager instead. Pulsar dashboard is deprecated.
 
 ## Install
 
+> **Note**
+> You can choose Docker installation（Method 1）, or directly download the source file and decompress it for installation（Method2），Method 1 and Method 2 can be chosen either.
```
Comment (Anonymitaet) [inline site2/docs/administration-pulsar-manager.md:15]:
```suggestion
To install Pulsar manager, you can choose one of the following methods.
```

File: site2/docs/administration-pulsar-manager.md [hunk 2]
```diff
@@ -70,7 +74,7 @@ spring.datasource.password=postgres
 ./gradlew build -x test
 ```
 
-### Enable JWT authentication
+### Enable JWT authentication[optional]
```
Comment (Anonymitaet) [inline site2/docs/administration-pulsar-manager.md:77]:
```suggestion
### Enable JWT authentication (optional)
```

File: site2/docs/administration-pulsar-manager.md [hunk 3]
```diff
@@ -49,9 +42,20 @@ cd ..
...
+```
+wget https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz
+tar -zxvf apache-pulsar-manager-0.2.0-bin.tar.gz
+cd pulsar-manager
+tar -zxvf pulsar-manager.tar
+cd pulsar-manager
+cp -r ../dist ui
+```
+
+### Use custom databases[optional]
```
Comment (Anonymitaet) [inline site2/docs/administration-pulsar-manager.md:56]:
```suggestion
### Use custom databases (optional)
```

**Pull request merged**